config/laravel-fulltext.php

```diff
@@ -2,15 +2,21 @@
 
 return [
 
-  'weight' => [
-    'title' => 1.5,
-    'content' => 1,
-  ],
+    /************************
+     * The database connection to be used
+     * This will use the db connection from config/database.php
+     */
+    'db_connection' => env('DB_CONNECTION'),
```

Comment:
This is a potential breaking change and/or it can lead to strange behaviour. Someone might use a different environment variable or has a fixed value for `config('database.default')`. If we default this to `null` and retrieve the value using `config('laravel-fulltext.db_connection') ?? config('database.default')` we make sure we use the same value Laravel uses now, while still allowing a custom connection.